src/lib.rs

```diff
@@ -452,6 +452,12 @@ impl LineEventHandle {
         }
     }
 
+    pub fn get_value(&self) -> Result<u8> {
```

Comment:
I'm kind of surprised the ioctl is implemented on this fd in the kernel, but it is there.  Seems like kind of an odd access pattern given that actual processing of events is out of the queue, but I'm sure just having that one fd at a certain point of execution probably make a few people's lives much easier.